src/Compilers/Core/CommandLine/BuildProtocol.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
The only intended consumers of this package are us and the sdk repo. Both of us have nullable on by default. Do we need these to be here?